simple-bills-gui/src/app/content/assets/assets.component.html

```diff
@@ -0,0 +1,316 @@
+<div class="container p-1 text-center">
+  <button class="btn btn-lg btn-outline-secondary m-2" (click)="openDepositCreationWindow(depositCreation)">
+    Add deposit
+  </button>
+  <br>
+  <hr>
+  <div class="row">
+    <div *ngIf="depositService.deposits$ | async as deposits else noData">
+      <table class="table table-striped align-middle">
+        <thead>
+        <tr>
+          <th>#</th>
+          <th>Deposit type</th>
+          <th>Deposit name</th>
+          <th>Deposit value</th>
+          <th>Bank name</th>
+          <th>Duration [months]</th>
+          <th>Annual interest rate</th>
+          <th></th>
+          <th></th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr *ngFor="let deposit of deposits; index as i">
+          <th scope="row">
+            {{i + 1}}
+          </th>
+          <td>
+            {{deposit.depositType}}
+          </td>
+          <td>
+            {{deposit.name}}
+          </td>
+          <td>
+            {{deposit.value | currency}}
+          </td>
+          <td>
+            {{deposit.bankName}}
+          </td>
+          <td>
+            {{deposit.durationInMonths }}
+          </td>
+          <td>
+            {{deposit.annualInterestRate}}%
+          </td>
+          <td>
+            <button
+              type="button"
+              class="btn btn-outline-secondary btn-sm"
+              aria-label="Close"
+              (click)="openDepositEditWindow(deposit, editDeposit)"
+            >Edit
+            </button>
+          </td>
+          <td>
+            <button
+              type="button" class="btn-close" aria-label="Close"
+              (click)="openDepositDeletionConfirmationWindow(deposit.name, deletionConfirmationWindow)"
+            >
+            </button>
+          </td>
+        </tr>
+        <tr class="bg-info fw-bold">
+          <th></th>
+          <td></td>
+          <td>Total limit:</td>
+          <!--            <td>{{totalLimit | currency}}</td>-->
+          <td></td>
+          <td></td>
+          <td></td>
+          <td></td>
+          <td></td>
+          <td></td>
+        </tr>
+        </tbody>
+      </table>
+    </div>
+  </div>
+</div>
+
+<!--
+      add new deposit modal window
+-->
+<ng-template #depositCreation let-modal>
+  <div class="modal-header">
+    <h4 class="modal-title" id="modal-deposit-creation">Deposit creation</h4>
+    <button type="button" class="btn-close" aria-label="Close" (click)="modal.dismiss('Cross click')"></button>
+  </div>
+  <div class="modal-body text-dark">
+    <form #createDepositForm="ngForm">
```

Comment:
Tak informacyjnie bardziej: u mnie wszystkie formularze były robione przez reactive-forms, wtedy ma się dynamiczne formularze (dodawanie pól do formularza, usuwanie z formularza), i dostęp do kontrolek bezpośrednio.

Takie statyczne ngForm jest do templatek, czyli jak masz staly formularz, a możemy sobie wyobrazić np. formularz który rozszerza się jak np. zamawiasz telewizor na allegro i ustawiasz adres jako niestandardowy (nie z listy), to się wtedy cały formularz rozszerza o to pole, a znika pole z wyborem z listy przykładowo. Wtedy reactive forms jest do użycia.

Ale tutaj ten jest też ok